### PR TITLE
Add constants and other setup for the orca database in a separate orca module

### PR DESCRIPTION
**Pull request by himanshunaidu**

### CODE REVIEW

File: src/transit_equity/orca/query/__init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+"""
+This package stores common queries used on the orca_ng database
```
Comment (bhazelton) [inline src/transit_equity/orca/query/__init__.py]: It looks like this is under `orca` not `orca_ng`, so I'm confused by this comment.

**Review by bhazelton — COMMENTED** [review]
Overall I think this looks good, I just had one confusion about a docstring.

File: src/transit_equity/orca/query/__init__.py (hunk 1 above)
Comment (himanshunaidu) [reply src/transit_equity/orca/query/__init__.py]: Oh that's incorrect. Thank you!

Comment (himanshunaidu) [reply src/transit_equity/orca/query/__init__.py]: Fixed!

**Comment by himanshunaidu** [comment]
Done!

**Review by bhazelton — APPROVED** [review]
Looks reasonable to me.

Not a big deal, but it looks like you're using triple single quotes for docstrings, while the numpy style is to use triple double quotes.

**Comment by himanshunaidu** [comment]
Ohh, I see! I missed that.
We do have a TLC session going on, so I guess I can make the required changes, in a separate PR.

**Comment by himanshunaidu** [comment]
Merging for now. Will raise a separate PR for changing the quotation marks.

**Pull request merged**